src/components/Dashboard.re

```diff
@@ -0,0 +1,328 @@
+  let latestBlockDate = momentWithUnix(secTimestamp) |> Moment.format("MMM DD, YYYY.");
+  let latestBlockTime = momentWithUnix(secTimestamp) |> Moment.format("HH:mm");
+  let transactions_total_xtz = Js.Float.toFixedWithPrecision(Utils.convertFromUtezToTez(transinfo.sumAmount), 2);
+  let fundraiserPercent = Js.Math.round((float_of_string(blockinfo.totalFundraiserCount) /. 30317.0) *. 100.0);
+  let sumFee = Js.Float.toFixedWithPrecision(Utils.convertFromUtezToTez(blockinfo.sum_fee), 2);
+  let consumedGas = Js.Float.toFixedWithPrecision(Utils.convertFromUtezToTez(blockinfo.sum_consumed_gas), 2);
+
+  let tez_staked = Js.Float.toFixedWithPrecision(Utils.convertFromUtezfToTez(blockinfo.bakers_sum_staking_balance), 2);
+  let total_tez = Js.Float.toFixedWithPrecision(Utils.convertFromUtezfToTez(blockinfo.totalTez), 2);
+  let percent_staked = Js.Math.round((blockinfo.bakers_sum_staking_balance /. blockinfo.totalTez) *. 100.0);
+  let periodKind = items##period_kind;
+
+  let rightTitle = switch periodKind {
+  | "proposal" => "PROPOSAL"
+  | "testing"=> "TESTING"
+  | "testing_vote" => "EXPLORATION"
+  | "promotion_vote" => "PROMOTION VOTE"
+  | _ => ""
+  };
+
+  <div className=Styles.container>
+    <div className=Styles.mainContainer>
+      <div className=Styles.leftContainer>
+        <div className=Styles.leftTopContainer(theme)>
+          {ReasonReact.string("Greetings! The Tezos ")}
+          <div className=Styles.networkContent(theme)>{ReasonReact.string(network)}</div>
+          {ReasonReact.string(" is now in cycle ")}
+          <div className=Styles.content1(theme)>{ReasonReact.string(string_of_int(items##meta_cycle))}</div>
+          <br />
+          {ReasonReact.string("Within this cycle, ")}
+          <div className=Styles.content1(theme)>{ReasonReact.string(string_of_int(blockinfo.blockCount) ++ " of " ++ string_of_int(blocksPerCycle))}</div>
+          {ReasonReact.string(" blocks ")}
+          <div className=Styles.content1(theme)>{ReasonReact.string("(" ++ Js.Float.toString(percentBaked) ++ "%)")}</div>
+          {ReasonReact.string(" have been baked.")}
+          <br />
+          {ReasonReact.string("The latest block ")}
+          <div className=Styles.content1(theme)>
+            {ReasonReact.string(items##hash)}
+            <CopyContent isReverse={true} />
+          </div>
+          {ReasonReact.string(" at level ")}
+          <div className=Styles.content1(theme)>{ReasonReact.string(string_of_int(items##level))}</div>
+          {ReasonReact.string(" was baked by ")}
+          <div className=Styles.content1(theme)>
+            {ReasonReact.string(items##baker)}
+            <CopyContent isReverse={true} />
+          </div>
+          {ReasonReact.string(" at ")}
+          <div className=Styles.content1(theme)>{ReasonReact.string(latestBlockTime)}</div>
+          {ReasonReact.string(" on ")}
+          <div className=Styles.content1(theme)>{ReasonReact.string(latestBlockDate)}</div>
```

Comment:
This date format has a period at the end, the period should be outside of the box.